app_feup/lib/view/Widgets/info_desk_card.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'generic_card.dart';
+
+Container h1(String text, BuildContext context, {bool initial = false}) {
+  final double marginTop = initial ? 15.0 : 30.0;
+  return Container(
+      margin: EdgeInsets.only(top: marginTop, bottom: 0.0, left: 20.0),
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: Text(text,
+            style:
+                Theme.of(context).textTheme.headline4.apply(fontSizeDelta: 0)),
+      ));
+}
+
+Container h2(String text, BuildContext context) {
+  return Container(
+      margin: const EdgeInsets.only(top: 13.0, bottom: 0.0, left: 20.0),
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: Text(text,
+            style:
+                Theme.of(context).textTheme.headline4.apply(fontSizeDelta: -4)),
+      ));
+}
+
+Container infoText(String text, BuildContext context, {bool last = false}) {
+  final double marginBottom = last ? 8.0 : 0.0;
+  return Container(
+      margin: EdgeInsets.only(top: 8, bottom: marginBottom, left: 20.0),
+      child: Align(
+        alignment: Alignment.centerLeft,
+        child: InkWell(
+          child: Text(
+            text,
+            style: Theme.of(context).textTheme.headline3,
+          ),
+        ),
+      ));
+}
```

Comment:
I think it was a great idea to create these functions and reuse them in every card! However, looking at the code's organization, I think it would make more sense if these functions were defined in the [useful_contacts_page](https://github.com/NIAEFEUP/project-schrodinger/blob/feature/useful-contacts-card/app_feup/lib/view/Pages/useful_contacts_card_page_view.dart) (outside the class definition) instead of in the infodesk card file since it's the page that is the "parent" widget of the other cards.